src/librustc_codegen_ssa/back/write.rs

```diff
@@ -517,9 +500,6 @@ fn copy_all_cgu_workproducts_to_incr_comp_cache_dir(
         if let Some(ref path) = module.bytecode {
             files.push((WorkProductFileKind::Bytecode, path.clone()));
         }
```

Comment:
The bitcode is always stored in the object file, right? That means that storing it separately in the incremental cache is not necessary anymore.

Reply:
Nice catch! I have removed this code and there were a bunch of follow-up clean-ups, including the removal of `WorkProductFileKind`.